<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Config</class>
 <widget class="QWidget" name="Config">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>440</width>
    <height>210</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>440</width>
    <height>210</height>
   </size>
  </property>
  <property name="maximumSize">
   <size>
    <width>440</width>
    <height>210</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Configuration</string>
  </property>
  <widget class="QLabel" name="lbl_mediaplayer">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>40</y>
     <width>91</width>
     <height>16</height>
    </rect>
   </property>
   <property name="text">
    <string>Media Player</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_player">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>40</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>MPV</string>
    </property>
   </item>
   <item>
    <property name="text">
     <string>VLC</string>
    </property>
   </item>
  </widget>
  <widget class="QLabel" name="lbl_discord">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>80</y>
     <width>141</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Share status to Discord</string>
   </property>
  </widget>
  <widget class="QCheckBox" name="cbox_discord">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>80</y>
     <width>82</width>
     <height>20</height>
    </rect>
   </property>
   <property name="text">
    <string>Yes</string>
   </property>
   <property name="checked">
    <bool>true</bool>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_reset">
   <property name="geometry">
    <rect>
     <x>170</x>
     <y>160</y>
     <width>141</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Reset Configuration</string>
   </property>
  </widget>
  <widget class="QPushButton" name="btn_deleteHistory">
   <property name="geometry">
    <rect>
     <x>330</x>
     <y>160</y>
     <width>101</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Delete History</string>
   </property>
  </widget>
  <widget class="QLabel" name="lbl_mirror">
   <property name="geometry">
    <rect>
     <x>40</x>
     <y>120</y>
     <width>56</width>
     <height>17</height>
    </rect>
   </property>
   <property name="text">
    <string>Mirror</string>
   </property>
  </widget>
  <widget class="QComboBox" name="combo_mirror">
   <property name="geometry">
    <rect>
     <x>220</x>
     <y>120</y>
     <width>121</width>
     <height>21</height>
    </rect>
   </property>
   <item>
    <property name="text">
     <string>uservideo</string>
    </property>
   </item>
  </widget>
  <widget class="QPushButton" name="btn_save">
   <property name="geometry">
    <rect>
     <x>20</x>
     <y>160</y>
     <width>131</width>
     <height>30</height>
    </rect>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
